Doc/Tutorial/chapter_seq_objects.tex

```diff
@@ -32,41 +32,40 @@ \section{Sequences and Alphabets}
 Alphabet()
 \end{minted}
 
-However, where possible you should specify the alphabet explicitly when creating your sequence objects - in this case an unambiguous DNA alphabet object:
+However, where necessary you can specify the alphabet is DNA:
```

Comment:
I guess this is a placeholder?

Reply:
Yes, we'll need somewhere to store the molecule type or whatever we're going to call it.

Reply:
So this section will be changed again before the next release?

Reply:
Yes - we need something for the SeqIO usage of molecule type at very least.

Reply:
OK let's revisit this later then.